Addon.Episerver.EnvironmentSynchronizer/Configuration/EnvironmentSynchronizerOptions.cs

```diff
@@ -13,7 +12,7 @@ public class EnvironmentSynchronizerOptions
 
         public bool RunInitializationModuleEveryStartup {  get; set;}
 
-        public IList<SiteDefinitionOptions> SiteDefinitions { get; set; }
+		public IList<SiteDefinitionOptions> SiteDefinitions { get; set; }
```

Comment:
Technically this is a breaking change, but since it can be considered an internal API, I think that we can ignore this.